Jenkinsfiles/Jenkinsfile.build_ami

```diff
@@ -0,0 +1,137 @@
+def private_key = 'dev-key'
```

Comment:
I agree, I'm in favor of using packer for this. Because I feared it might require heavy refactoring to the various playbooks and roles used in `build_ami`, I opted for this approach. At the moment, it's not _too_ painful to debug locally. The most important detail for running the `build_ami` playbook on your machine is having access to the Ansible vault password and a private key suitable for accessing the build instances. Unfortunately, I haven't had the chance to document how much of this wiring works. :-(